docs/man/man1/.files_generated

```diff
@@ -1,31 +0,0 @@
-kubectl-annotate.1
```

Comment:
Was this file used for anything?  It doesn't seem terribly useful, but I don't know it would have existed initially.